documentation/modules/adding-source-provider.adoc

```diff
@@ -199,3 +199,12 @@ ifdef::ova[]
 An error message might appear that states that an error has occurred. You can ignore this message.
 ====
 endif::[]
+
+ifdef::vmware,rhv,ostack,cnv,cnv2[]
+. Optional: To add access to the UI of the provider:
```

Comment:
```suggestion
. Optional: Add access to the UI of the provider:
```
We are advised by the IBM SG to use full intro sentences :-) Even if you have further substeps, it ok to have it this way.

Reply:
Fixed.